coding/fuzz/fuzz_targets/reed_solomon.rs

```diff
@@ -45,14 +45,12 @@ fn fuzz(input: FuzzInput) {
     };
     assert_eq!(decoded, payload, "decode with all chunks failed");
 
-    if min as usize <= chunks.len() {
-        let subset: Vec<Chunk<Sha256>> = chunks.into_iter().take(min as usize).collect();
-        let decoded_subset = match decode::<Sha256>(total, min, &root, subset) {
-            Ok(data) => data,
-            Err(e) => panic!("decode with min chunks failed: {e:?}"),
-        };
-        assert_eq!(decoded_subset, payload);
-    }
+    let subset: Vec<Chunk<Sha256>> = chunks.into_iter().take(min as usize).collect();
```

Comment:
May be a good idea to shuffle before taking the min (to ensure we mix/match original + recovery shards)

Reply:
Once this is done, should be g2g